lib/config.js

```diff
@@ -42,110 +37,23 @@ cc('views.bundle.paths.input.root', () => path.join(
   config.paths.cache, 'bedrock-views', 'root.js'));
 // common paths
 cc('views.bundle.paths.output.local', () => path.join(
-  config.paths.cache, 'bedrock-views', 'static'));
+  config.paths.cache, 'bedrock-views', config.views.bundle.mode));
 cc('views.bundle.paths.output.public', () => '/static');
 // normal paths
 cc('views.bundle.paths.output.main', () => path.join(
   config.views.bundle.paths.output.local, 'js', 'bundle.js'));
-// optimized paths
-cc('views.bundle.paths.output.optimize.main', () => path.join(
-  config.views.bundle.paths.output.local, 'js', 'bundle.min.js'));
 
-// route for optimized js
+// route for output js
 c.pushComputed('express.static', () => ({
   route: config.views.bundle.paths.output.public,
   path: config.views.bundle.paths.output.local
 }));
 
-// TODO: consider adding feature to read less vars (like paths)
-// from package files bend toward zero config
-const componentsUrl = config.views.modules.baseURL;
-
-// TODO: refactor some of these config settings into other modules
-
-// branding config
-config.views.brand = {};
-config.views.brand.name = '';
-
-// TODO: backwards compatibility, remove brand at top level of config
-config.brand = config.views.brand;
-
 config.views.cache = false;
+// and consolidate.js engine can be used as long as proper lib is installed.
```

Comment:
Are these two lines related?  How is lodash a views engine?

Reply:
Typo there: s/and/any/.  consolidate.js wraps lots of template engines.  lodash templates are one choice.  Not as featureful as swig or similar but seemed a ok choice if we just pass in a simple var.